app/views/schools/participants/index.html.erb

```diff
@@ -5,50 +5,91 @@
-        <% unless current_user.mentor? %>
-          <li><%= govuk_link_to "Add yourself as a mentor", add_schools_participants_path(cohort_id: @school_cohort.cohort.start_year, type: :self), class: "govuk-link--no-visited-state" %></li>
-        <% end %>
-      </ul>
-    </div>
-  </div>
-  <div class="govuk-grid-column-full">
-    <% if @categories.any? %>
-      <%= govuk_tabs(title: 'Contents') do |component| %>
-        <% if participants_active?(@categories.ect_categories) %>
-          <% component.tab(label: 'ECTs') do %>
-            <%= render partial: 'ects', locals: { ect_profiles: @categories.ect_categories } %>
-          <% end %>
+      <% if actions?(@participants) %>
+        <% if @participants.orphan_ects.any? %>
+          <%= govuk_warning_text(text: "You need to assign a mentor to these ECTs.") %>
+          <ul class="govuk-list">
+            <% @participants.orphan_ects.sort_by(&:participant_full_name).each do |ect| %>
+              <li>
+                <%= govuk_link_to(ect.participant_full_name, schools_participant_path(id: ect.participant_profile_id,
+                                                                                      school_id: @school),
+                                  no_visited_state: true,
+                                  classes: ["govuk-\!-padding-bottom-static-1 govuk-\!-margin-bottom-static-0"]) %>
+              </li>
+            <% end %>
+          </ul>
+          <br>
         <% end %>
+      <% end %>
+
+      <% @participants.mentors.keys.sort_by(&:participant_full_name).each do |mentor| %>
+        <div class="group-card">
+          <%= govuk_summary_list(actions: false, borders: nil) do |list| %>
+            <% list.row do |row| %>
+              <% row.key(text: "Mentor") %>
+            <% end %>
+
+            <% list.row do |row| %>
+              <% row.key(text: govuk_link_to(mentor.participant_full_name,
+                                             schools_participant_path(id: mentor.participant_profile_id,
+                                                                      school_id: @school),
+                                             no_visited_state: true),
+                         classes: ["govuk-!-font-weight-regular"]) %>
+              <% row.value(text: render(ParticipantStatusTagComponent.new(profile: mentor.participant_profile,
+                                                                          induction_record: mentor,
+                                                                          has_mentees: true))) %>
+            <% end %>
 
-        <% if participants_active?(@categories.mentor_categories) %>
-          <% component.tab(label: 'Mentors') do %>
-            <%= render partial: 'mentors', locals: { mentor_profiles: @categories.mentor_categories } %>
+            <% list.row do |row| %>
+              <% row.key(text: "ECTs") %>
```

Comment:
Same here, probably should be a heading.

Reply:
Done